src/requirements.txt

```diff
@@ -1,6 +1,6 @@
 argparse==1.2.1
 # we're stuck on 3.0 until we get rid of the xenial stragglers
```

Comment:
nit: remove the comment too?

Not sure what to do about xenial though, my guess is that it might just need to be dropped entirely to make this work?